src/game/Utils/Cinematics.cc

```diff
@@ -172,7 +172,8 @@ catch (...) { return 0; }
 
 void SmkCloseFlic(SMKFLIC* const sf)
 {
-	SmackClose(sf->SmackerObject);
+	if(sf->SmackerObject)
+		SmackClose(sf->SmackerObject);
```

Comment:
I think this would better fit into SmackClose to mimic free/delete, but then again, this is its only user, so it's not that important.

Reply:
right... will change it

Reply:
rebased and pushed this change